style.scss

```diff
@@ -63,6 +63,23 @@ body {
 .todo-item__label {
     flex: 1;
 }
+.todo-item__edit-input {
+    flex: 1;
+    font-size: 1em;
+    display: none;
+}
+// Todo item edit state
+.todo-item.editing {
```

Comment:
This should follow BEM. `.todo-item--editing` maybe?

Reply:
Good point. I wasn't sure if I should use BEM here since .editing is an interaction state which deals with JS rather than an alternate presentation style. Good to know

I will refactor to use BEM.

Reply:
Javascript is only for things where you are trying to select something with jquery. This is a style state where you are displaying differently during edit.